iothub/device/samples/getting started/MessageReceiveSample/MessageReceiveSample.cs

```diff
@@ -48,10 +48,10 @@ public async Task RunSampleAsync()
             Console.WriteLine($"{DateTime.Now}> Use the Azure Portal IoT hub blade or Azure IoT Explorer to send a message to this device.");
             await ReceiveC2dMessagesPollingAndCompleteAsync(cts.Token);
 
-            if (_transportType != TransportType.Http1)
+            if (_transport != Transport.Http1)
```

Comment:
We dropped support for HTTP on the device client, so you can remove this code block.